### ISSUE TITLE
AddCloudfoundryActuators throws exception when configured to use "/cloudfoundryapplication" as path

### ISSUE THREAD

**Issue by hananiel** [SteeltoeOSS/Steeltoe#501]
When configured either using the services extension or host builder extension, this feature automatically sets the path to "/cloudfoundryapplication". However when you specify this path explicitly it throws an error. Need a more user friendly experience. 
```Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

/cloudfoundryapplication HTTP: Get
/cloudfoundryapplication HTTP: Get
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, CandidateState[] candidateState)
```

**Comment by jkonicki** [comment]
@hananiel What version does this effect?  Please add to the Labels.

**Comment by hananiel** [comment]
Verified fixed in 3.2.1
